packages/perspective-jupyterlab/src/ts/view.ts

```diff
@@ -37,11 +52,32 @@ function isEqual(a, b) {
     return true;
 }
 
+// proxy_table assigns itself table's prototype which means it has all
+// of table's methods, but Typescript doesn't see that
+type ProxyTable = proxy_table & Table;
+
 /**
  * `PerspectiveView` defines the plugin's DOM and how the plugin interacts with
  * the DOM.
  */
 export class PerspectiveView extends DOMWidgetView {
+    luminoWidget: PerspectiveJupyterWidget;
+    perspective_client: PerspectiveJupyterClient;
+    _pending_binary?: unknown;
```

Comment:
these values are only ever assigned, and not read, so I think it's safe to leave them unknown